libs/ui-components/src/components/Fleet/CreateFleet/steps/UpdateConfirmChangesModal.tsx

```
@@ -0,0 +1,85 @@
+const getModalContent = (setting: StepSetting, t: TFunction) => {
+
+  switch (setting) {
+    case 'all-settings':
+      return {
+        title: t('Switch to basic update configuration?'),
+        primaryAction: t('Switch to basic configurations'),
+        question: t('Are you sure you want to switch to basic update configurations?'),
+        result: t('If you confirm, all advanced configurations will be lost.'),
+      };
+    case 'rollout-policies':
+      const policyTitle = t("Don't use rollout policies configurations");
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)